incubator/hydra/templates/hydra_deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      name: {{ template "fullname" . }}
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+    spec:
+      volumes:
+        - name: hydra-data
+        {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ template "fullname" . }}
+        {{- else }}
+          emptyDir: {}
+        {{- end }}
+        - name: hydra-secret
+          secret:
+            secretName: {{ template "fullname" . }}-secret
+      containers:
+        - name: hydra
+          image: {{ .Values.image }}:{{ .Values.imageTag }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: ["hydra", "host", "--dangerous-auto-logon"]
```

Comment:
How do you propose we do the sql migrations beforehand?

We can run 2 commands by doing

```
/bin/sh -c '/go/bin/hydra migrade sql $DATABASE_URL && /go/bin/hydra host'
```

but I'm not a huge fan of that solution.

A better solution in my opinion would be init containers.

https://kubernetes.io/docs/concepts/workloads/pods/init-containers/

so I'll try to set that up.